feedback/migrations/0020_remove_feedbacktag_feedbacks_conversation_and_more.py

```diff
@@ -29,12 +58,27 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='feedback',
             name='conversation',
-            field=models.ForeignKey(default=None, on_delete=django.db.models.deletion.PROTECT, related_name='feedbacks', to='feedback.conversation'),
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.PROTECT, related_name='feedbacks', to='feedback.conversation'),
             preserve_default=False,
         ),
+        # Go through all existing feedback, create conversations and
+        # link feedback to correct conversations
+        migrations.RunPython(generate_conversations_for_old_feedback, delete_conversations),
```

Comment:
All you have to do is set the conversations to None for all feedbacks in `delete_conversations` before deleting the converstations.

Reply:
I didn't try reversing the migration, since I'm not sure how to do that when running the program in a container. Good point, and thanks for the tip!
Although I do wonder, is it necessary to actually delete the Conversations in the reverse_code function of the RunPython, as if the migration would be reversed, the field pointing to the model from Feedback and entire model Conversation would be deleted, so there would be no more Conversations. Would it actually be better style to use [noop](https://docs.djangoproject.com/en/4.0/ref/migration-operations/#django.db.migrations.operations.RunPython.noop) instead as the reverse function?

Reply:
Yes, noop would be better if it works. I also don't see why it wouldn't.

Reply:
I fixed these and tested reversing the migration (there was a typo that also needed to be fixed), it now worked (at least when quickly testing)